test/testmatrix.cpp

```diff
@@ -653,15 +682,16 @@ TEST(Matrix, illegalLinearEqSetZeroDimension)
 
 TEST(Matrix, simpleSymbolDet)
 {
```

Comment:
Please don't alter an existing unit test that ensures everything keeps on working as expected (though it might fail when a bug related to the det() method is fixed, but that's also worth to be noticed about). Write a new one instead, it's only a couple of lines of code.